samples/NativeIISSample/Properties/launchSettings.json

```diff
@@ -14,8 +14,8 @@
       "commandLineArgs": "$(IISExpressArguments)",
       "environmentVariables": {
         "IIS_SITE_PATH": "$(MSBuildThisFileDirectory)",
-        "ANCM_PATH": "$(AncmPath)",
-        "ANCMV2_PATH": "$(AncmV2Path)",
+        "ANCM_PATH": "$(AspNetCoreModuleV1ShimDll)",
```

Comment:
Revert or do it for all launchSettings.json

Reply:
This was a missed file with https://github.com/aspnet/IISIntegration/commit/b0deed05be7710d80e28a1ce95ff4e760939644b